code/game/objects/items/weapons/species_enhancement/species_enhancement.dm

```diff
@@ -0,0 +1,47 @@
+/obj/item/weapon/species_enhancement
+	name = "Species enhancer"
+	desc = "You shouldn't see this."
+	icon = 'icons/obj/hypo.dmi'
+	item_state = "syringe_0"
+	icon_state = "lepopen"
+	var/used = null
+	var/list/species = null
+
+/obj/item/weapon/species_enhancement/update_icon()
+	if(used)
+		icon_state = "lepopen0"
+	else
+		icon_state = "lepopen"
+
+/obj/item/weapon/species_enhancement/attack(mob/M as mob, mob/user as mob)
+	if(!M || !user)
+		return
+
+	if(!ishuman(M) || !ishuman(user))
+		return
+
+	if(src.used)
+		return
+
+	if(ishuman(M))
```

Comment:
kill this whole block off and replace it with `if(!species.Find(M.get_species()))`